README.md

```diff
@@ -185,20 +185,20 @@ Nano ID could be started from number. HTML ID can’t be started from the numbe
 
 ### React Native
 
-To generate secure random IDs in React Native, you must use
-[a native random generator] and asynchronous API:
+To generate secure random IDs in React Native, you must use a native random 
+generator (like [expo-random] or [react-native-securerandom]) and the
+asynchronous API:
 
-```js
-const generateSecureRandom = require('react-native-securerandom').generateSecureRandom
-const format = require('nanoid/async/format')
-const url = require('nanoid/url')
+```ts
+import { getRandomBytesAsync } from 'expo-random';
+import format from 'nanoid/async/format';
+import url from 'nanoid/url';
```

Comment:
All examples have no semicolons. It will be strange for users to see different style guides in one doc.

Reply:
Also, note that your ES import syntax is incorrect. ES imports require a full path. This imports will work only in Babel. It was a reason why I used `require` in all other docs. If you want to change `require` to `import` you need to do it across all docs (and first start with good reason of doing it)